### DISCUSSION TITLE
Unable to instantly display table data, Please help

### DISCUSSION THREAD

**Discussion by idofarit** [supabase/supabase#23223] [Questions]
I am working with a react project. As I add new table row to my table, it wont show in the display instantly until manually refreshed. Please help me out how to show my newly added table row data as soon as I submit.

**Comment by GaryAustin1** [comment]
You asked the same question yesterday and did not get an answer.  You need to provide more info on what you are doing and what data you want to reflect.  Sort of sounds like you are asking a react question if you are just trying to update your screen as you do uploads from that client.  If you want your data to reflect all updates on a remote database then you have to poll or use realtime.
